api/c/tests/unit/bingo-nosql/bingo_nosql_test.cpp

```diff
@@ -32,6 +32,7 @@ TEST(BingoNosqlTest, test_enumerate_id)
     int e = bingoEnumerateId(db);
     while (bingoNext(e))
     {
+        ASSERT_EQ(count, bingoGetCurrentId(e));
```

Comment:
Why we may rely upon the Id of the object in the enumerator must be equal to the sequential index?
This is nowhere specified.